circuit/environment/Cargo.toml

```diff
@@ -59,6 +59,11 @@ version = "0.2"
 [dependencies.once_cell]
 version = "1.18.0"
 
+[dependencies.sha2]
```

Comment:
Can you explain why this crate is safe, or the safest option for SHA2

Reply:
we're already using it: https://github.com/ProvableHQ/snarkVM/blob/595d80778f24d15cedead12f4b001d94f09cc502/algorithms/Cargo.toml#L140